authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

```diff
@@ -0,0 +1,19 @@
+[id="cco-mode-manual"]
```

Comment:
Manual mode overview taken from the former CCO page.

Reply:
https://deploy-preview-29545--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-manual.html